.github/workflows/ci.yml

```diff
@@ -16,9 +16,8 @@ jobs:
       fail-fast: false
       matrix:
        include:
-         # Rails 6.1 builds >= 2.5
+         # Rails Master builds >= 2.5
          - ruby: ruby-3.0.0-preview1
-           allow_failure: true
```

Comment:
If we're adding Rails master builds as well as rc builds master should be allowed to fail.

Reply:
Ok